main.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+// 各種DOMを取得
+const $homeContainer = document.querySelector('#home-container');
+const $questionContainer = document.querySelector('#question-container');
+const $main = document.querySelector('#main');
+const $para = document.querySelector('#para');
+const $btn = document.querySelector('#btn');
+const $quizTitle = document.querySelector('#quiz-title');
+const $quizGenre = document.querySelector('#quiz-genre');
+const $quizDif = document.querySelector('#quiz-dif');
+const $quizQuestion = document.querySelector('#question');
+const $quizAnswer0 = document.querySelector('#answer0');
+const $quizAnswer1 = document.querySelector('#answer1');
+const $quizAnswer2 = document.querySelector('#answer2');
+const $quizAnswer3 = document.querySelector('#answer3');
+const $quizAnswerBtn = document.querySelectorAll('#question-container > button');
+
+// ボタンを押下すると実行する非同期処理
+$btn.addEventListener('click', () => {
+  $main.innerText = '取得中';
+  $para.innerText = '少々お待ちください';
+  $btn.classList.add('transparent');
+  getAPI();
+});
+
+// 正解数用のカウンター
+let correctCounter = 0;
+
+// 何問目か用のカウンター
+let quizCounter = 0;
+
+// クイズを出題するクラス
+class CreateQuiz {
```

Comment:
Class 名は `名詞` にした方がわかりやすいです。

```suggestion
class Quiz {
```